projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -494,7 +497,7 @@ public function add_woocommerce_installation_menu() {
 		 * @since 10.3.0
 		 * @param bool $jetpack_show_wpcom_woocommerce_installation_menu Load the WordPress.com WooCommerce Installation menu item. Default to false.
 		 */
-		if ( apply_filters( 'jetpack_show_wpcom_woocommerce_installation_menu', false ) ) {
+		if ( apply_filters( 'jetpack_show_wpcom_woocommerce_installation_menu', false, $current_product ) ) {
```

Comment:
Looks like that filter docs need an update to account for the variable?

Reply:
Updated. Please let me know if that looks right. I wasn't entirely sure if @since should be updated to reflect the change, or if it stood for the filter's initial introduction.

Reply:
If it follows Core then it's in reference to when it was first introduced. [Docs standards allow for additional since tags to create a changelog](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#since-section-changelogs) but I'm not sure we need that here. 

Has [WordPress phpcs](https://github.com/WordPress/WordPress-Coding-Standards) flagged variable and description alignments at all? Not sure if it picks that up.

Reply:
> If it follows Core then it's in reference to when it was first introduced. [Docs standards allow for additional since tags to create a changelog](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#since-section-changelogs) but I'm not sure we need that here.

👍 

> Has [WordPress phpcs](https://github.com/WordPress/WordPress-Coding-Standards) flagged variable and description alignments at all? Not sure if it picks that up.

It picks up a lot of things, but it's doesn't seem to have an opinion on that filter doc. 😅